docs/production/mobile-push-notifications.md

```diff
@@ -15,12 +15,13 @@ mobile apps.
 
 :::{important}
 
-These instructions are for Zulip Server 9.0+. If you are running an older
-version of Zulip ([check](https://zulip.com/help/view-zulip-version) if you are
-unsure), see the [Zulip 8.x
+These instructions are for Zulip Server 9.0+, and apply only if you did not
```

Comment:
i think this diff meant to also say Zulip Server 10.0+? I'll tweak this assuming that.

Reply:
No, the instructions are still for 9.0+, right? We haven't modified these instructions since then.